bubblewrap.c

```diff
@@ -389,18 +397,21 @@ static void
 drop_caps (void)
 {
   struct __user_cap_header_struct hdr;
-  struct __user_cap_data_struct data;
+  struct __user_cap_data_struct data[2];
```

Comment:
Odd that we use `memset` on `hdr`, but explicit assignment on `data` to zeros.  Maybe just do `= {0}` for both?